packages/rum-core/src/common/utils.js

```diff
@@ -200,13 +200,34 @@ function getTimeOrigin() {
   return PERF.timing.fetchStart
 }
 
-function getPageMetadata() {
+function getNetworkInformation() {
+  const connection = navigator && navigator.connection
+
+  if (!(connection && typeof connection === 'object')) {
```

Comment:
It's better to not negate the condition and just wrap the following return inside if.